DEV-7/DEV-7/Bmw.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+
+namespace DEVTask7
+{
+    /// <summary>
+    /// Class bmw extends product class
+    /// </summary>
+    [JsonObject]
+    class Bmw : Product
+    {
+        public Bmw()
+        {
+            Brand = "Bmw";
+        }
+
```

Comment:
too many arguments in constructor, think about join some car properties into separate classes